extensions/dicom-microscopy/src/components/ViewportOverlay/index.tsx

```diff
@@ -24,72 +24,66 @@ interface OverlayItem {
  * @param {*} extensionManager is used to load the image data.
  * @returns
  */
-export const generateFromConfig = ({
-  topLeft = [],
-  topRight = [],
-  bottomLeft = [],
-  bottomRight = [],
-  itemGenerator = () => {},
-}: {
-  topLeft?: OverlayItem[];
-  topRight?: OverlayItem[];
-  bottomLeft?: OverlayItem[];
-  bottomRight?: OverlayItem[];
-  itemGenerator?: (props: any) => any;
-}) => {
-  return (props: any) => {
-    const topLeftClass = 'top-viewport left-viewport text-primary-light';
-    const topRightClass = 'top-viewport right-viewport-scrollbar text-primary-light';
-    const bottomRightClass = 'bottom-viewport right-viewport-scrollbar text-primary-light';
-    const bottomLeftClass = 'bottom-viewport left-viewport text-primary-light';
-    const overlay = 'absolute pointer-events-none microscopy-viewport-overlay';
+export const generateFromConfig = ({ config, ...props }) => {
+  const {
+    topLeft = [],
+    topRight = [],
+    bottomLeft = [],
+    bottomRight = [],
+  }: {
+    topLeft?: OverlayItem[];
+    topRight?: OverlayItem[];
+    bottomLeft?: OverlayItem[];
+    bottomRight?: OverlayItem[];
+  } = config['MicroscopyViewportOverlay'] ?? {};
+  const topLeftClass = 'top-viewport left-viewport text-primary-light';
```

Comment:
Hmmm, config['MicroscopyViewportOverlay'] sounds odd, our customizations should not be based on appConfig, but rather through customizationService. See cornerstoneViewportOverlay on how we use it, and also here on how we consume it 

https://docs.ohif.org/platform/services/customization-service/customizationService

Reply:
This was the old customization - in the private mode, we can read the old customization configuration and apply it to the customization service, but then this extension should just read the configuration from the configuraiton service.  See how this is done within the OHIF overlay/customization setup.